snap/snapcraft.yaml

```diff
@@ -193,12 +205,16 @@ parts:
       VERSION=$( awk -F\" '/version_string/ { print $2 }' gen/multipass/version.h )
       craftctl set version=${VERSION}
       craftctl set grade=$( echo ${VERSION} | grep -qe '^[0-9]\+\.[0-9]\+\.[0-9]\+\($\|-rc\)' && echo stable || echo devel )
-      sed -i -e 's@^Icon=\(.\+\)$@Icon=/usr/share/icons/hicolor/scalable/apps/\1.svg@' ${CRAFT_PART_INSTALL}/usr/share/applications/multipass.gui.desktop
+      sed -i -e 's@^Icon=\(.\+\)$@Icon=/usr/share/icons/hicolor/scalable/apps/\1.svg@' ${CRAFT_PART_INSTALL}/usr/share/applications/multipass.desktop-gui.desktop
 
       cd ${CRAFT_PART_SRC}/3rd-party
       find . -name 'LICENSE*' -print0 | xargs -0 -I FILE install --mode=644 -D --no-target-directory FILE ${CRAFT_PART_INSTALL}/licenses/FILE
 
   qemu:
+    build-environment:
+    - LD_LIBRARY_PATH: ""
+    - XDG_DATA_DIRS: /root/stage/usr/share:/usr/share
+    - PKG_CONFIG_PATH: ""
```

Comment:
I am unsure why these are here.  I see @luis4a0 added these.  Could you please explain @luis4a0?  Thanks!